tests/test_changelog.py

```diff
@@ -800,6 +1080,16 @@ def test_generate_tree_from_commits(gitcommits, tags):
     assert tuple(tree) == COMMITS_TREE
 
 
+def test_generate_tree_from_commits_merge_prerelease(gitcommits, tags):
+    parser = defaults.commit_parser
+    changelog_pattern = defaults.bump_pattern
+    tree = changelog.generate_tree_from_commits(
+        gitcommits, tags, parser, changelog_pattern, merge_prerelease=True
+    )
+
+    assert tuple(tree) == COMMITS_TREE_MERGED_PRERELEASES
```

Comment:
Should `COMMITS_TREE_MERGED_PRERELEASES` be renamed to `COMMITS_TREE_AFTER_MERGED_PRERELEASES`? 

How is this test different than just `test_generate_tree_from_commits`?

Reply:
Yeah, that name is better. It's different in the sense, that `COMMITS_TREE_AFTER_MERGED_PRERELEASES` does not contain pre-releases (there were 2) and the changes of them are now part of the next non-prerelease.